app/pages/BuyNBTC/BuyNBTCTabContent.tsx

```diff
@@ -129,9 +126,10 @@ export function BuyNBTCTabContent() {
 				<ArrowDown className="text-primary justify-center w-full flex mb-2 p-0 m-0" />
 				<YouReceive isSuiWalletConnected={isSuiWalletConnected} mistAmount={mistAmount} />
 				{isSuiWalletConnected ? (
-					<Button type="submit" disabled={isPending} isLoading={isPending}>
+					<button className="btn btn-primary" type="submit" disabled={isPending}>
+						{isPending && <span className="loading loading-spinner"></span>}
```

Comment:
This is repeated. Let's have a helper component for that in /ui/components, and update other places where this is used.
```
function Spinner({isLoading}) {
  if (!isLoading) return
  return (<span className="loading loading-spinner" />)
}
```

Reply:
Done